### PR TITLE
[ADD] account: allow partial reconciliation with write-off

### PR DESCRIPTION
**Pull request by smetl**
Suppose you have an invoice of 3000. You want to pay with paypal
3 x 1000 but having of fee of 10 in each transaction. So, you attempt
to reconcile 3000 with 3 x (990 + 10) that is not possible with the
reconciliation widget.

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/account/static/src/js/reconciliation/reconciliation_model.js [hunk 1]
```diff
@@ -102,6 +102,10 @@ var StatementModel = BasicModel.extend({
         this.valuemax = 0;
     },
 
+    _round: function (value, digits) {
+        return Number(Math.round(value + 'e' + digits) + 'e-' + digits);
```
Comment (xmo-odoo) [inline addons/account/static/src/js/reconciliation/reconciliation_model.js]: Wouldn't it be less trashy to use `Math.pow` to shift the value back and forth around rounding?

Also beware, Math.round will round to +∞ in all cases, which is unlike pretty much anything else, and definitely unlike most (if not all) of the roundings done on the Python side.

**Comment by qdp-odoo** [comment]
branch landed on master, for saas11.3, at revision https://github.com/odoo/odoo/commit/b10932d8a51ae10d95b776dfcf4ea417ac5ec711

**Comment by wtaferner** [comment]
@qdp-odoo 
Can I try this to directly backport this as a hotfix my current issue?

**Comment by qdp-odoo** [comment]
@wtaferner 
Indeed, we detected that the former issue was already 'fixed' in master thanks to this commit and we're currently investigating to see if it's worth backporting it (partially or not).

**Comment by wtaferner** [comment]
@qdp-odoo 
After not hearing anything else I finally tried to simply cherry pick the commit https://github.com/odoo/odoo/commit/b10932d8a51ae10d95b776dfcf4ea417ac5ec711 you merged for saas11.3 and it resolves the issue. Unfortunately this is no official fix for the created OPW-815421.

By directly backporting without changes showed only one "issue" as the field ```computed_with_tax``` does not exist, but this is ignored by the ORM. So, we probably just need to remove the propagation to the backend and keep the anticipated logic just in the front end, but as said, the official solution seems to take too much time for me...as the issue is unresolved and it is impossible to reconcile the use case shown in the ticket and the customer is blocked...by now afaik it is still very complicated and even impossible to use taxes on ```account.move``` respectively ```account.move.line``` via UI .

Do you see any contradictions by now to just do it or are their any pitfalls and possible negative impacts on 11.0 in your opinion?

@nim-odoo 
FYI

**Pull request closed without merging**